docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 web:
-  image: node:8
+  image: jmervine/herokudev-node:8
```

Comment:
What's this change about?

Reply:
Just using a different base image for my local development environment. One that more closely matches the deployment target. This doesn't impact the application running on heroku.

Reply:
If you run docker, your can use with the following command:

```
docker-compose run --rm --service-ports web bash

# then setup with
npm install

# and run with
node app.js

# or run tests with
node make test
# ... etc
```